lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -257,6 +257,11 @@ class Chef::Knife::VsphereVmClone < Chef::Knife::BaseVsphereCommand
          description: 'Disable host key verification',
          boolean: true
 
+  option :node_ssl_verify_mode,
+         long: '--node-ssl-verify-mode [peer|none]',
+         description: 'Whether or not to verify the SSL cert for all HTTPS requests when bootstrapping',
+         boolean: true
```

Comment:
I think this line may be unnecessary since we're expecting a value to follow `--node-ssl-verify-mode`